src/main/scala/uk/gov/hmrc/zap/api/ZapScan.scala

```diff
@@ -66,6 +80,7 @@ class ZapScan(zapClient: ZapClient) extends Eventually {
   private def hasCallCompleted(path: String): Value = {
```

Comment:
Return type of `Value` is difficult to understand (too generic)

Reply:
It's also a bit confusing that `Run` is returned by methods that later return `Unit` so the value is not really used